CHANGES.rst

```diff
@@ -369,6 +369,9 @@ astropy.time
 - Time formats that do not use ``val2`` now raise ValueError instead of
   silently ignoring a provided value. [#9373]
 
+- Custom time formats (implemented in subclasses of ``TimeFormat``) now have
```

Comment:
This applies to all Time formats, not only "custom" ones.  I usually think of "custom" being user-defined.  The PR title should be updated as well.